examples/advance_music_cog.py

```diff
@@ -469,6 +469,32 @@ async def previous(self, ctx, index: int = None):
         if previous_player := await self.MusicManager.previous(ctx, index):
             await ctx.send(f"Rewinding from {previous_player[0].title}")
 
+    # Spotify song from user
+    @commands.command()
+    async def play_user_spotify(self, ctx, user: discord.Member = None):
+        user = user if user else ctx.guild.get_member(ctx.author.id)
+        spotify_result = next((activity for activity in user.activities if isinstance(activity, discord.Spotify)), None)
+
+        if spotify_result:
+            await ctx.send(f'{user.name} is not listening to Spotify.')
+            return
+        
+        query = f"{spotify_result.title} {spotify_result.artist}"
+
+        player = await self.MusicManager.create_player(query, user)
```

Comment:
The requester should not be the user, but the ctx.author.

Reply:
fixed